### PR TITLE
Editing dlgRenameSheet

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

File: instat/dlgRenameSheet.vb [hunk 1]
```diff
@@ -34,17 +34,14 @@ Public Class dlgRenameSheet
 
     Private Sub ReopenDialog()
 
-    End Sub
-
+    End Sub
+
     Private Sub InitialiseDialog()
-        'set the function
+        ucrInputNewName.SetName(ucrDataFrameToRename.cboAvailableDataFrames.SelectedItem)
```
Comment (dannyparsons) [inline instat/dlgRenameSheet.vb]: This is a default not for initialise

**Comment by Lunalo** [comment]
@dannyparsons  This now fixed

File: instat/dlgRenameSheet.Designer.vb [hunk 2]
```diff
@@ -22,48 +22,48 @@ Partial Class dlgRenameSheet
...
         'lblName
         '
         Me.lblName.AutoSize = True
-        Me.lblName.Location = New System.Drawing.Point(145, 21)
+        Me.lblName.Location = New System.Drawing.Point(207, 9)
         Me.lblName.Name = "lblName"
         Me.lblName.Size = New System.Drawing.Size(60, 13)
-        Me.lblName.TabIndex = 6
+        Me.lblName.TabIndex = 1
         Me.lblName.Tag = "Name"
```
Comment (dannyparsons) [inline instat/dlgRenameSheet.Designer.vb]: The tag should also be "New_Name" so that it displays correctly.

File: instat/dlgRenameSheet.vb [hunk 3]
```diff
@@ -27,24 +27,19 @@ Public Class dlgRenameSheet
...
     Private Sub InitialiseDialog()
-        'set the function
         ucrBase.clsRsyntax.SetFunction(frmMain.clsRLink.strInstatDataObject & "$rename_dataframe")
-
-
     End Sub
 
     Private Sub SetDefaults()
-
+        ucrInputNewName.SetName(ucrDataFrameToRename.cboAvailableDataFrames.SelectedItem)
```
Comment (dannyparsons) [inline instat/dlgRenameSheet.vb]: Can you also add the default that ucrInputNewName starts with focus?

**Comment by Lunalo** [comment]
You can now have a look at this

**Comment by dannyparsons** [comment]
Can you also add that when the data frame is changed the current name is added to the Input text and focus is set to there?

**Comment by Lunalo** [comment]
So you mean I should also do 
 ucrInputNewName.SetName(ucrDataFrameToRename.cboAvailableDataFrames.SelectedItem)
 ucrInputNewName.Focus()
in dataframeChanged event?
I think i get your point on this

**Comment by dannyparsons** [comment]
Yes so that the Input box changes to the current name when Data frame is
changed.

**Comment by Lunalo** [comment]
I think have now done that

**Pull request merged**